### PR TITLE
Adds information on Adb installation for windows users

### LINKED ISSUE #189: Complete ADB installation in Windows
ADB tool will not be installed by default within Android Studio as it is written in the contributor's guide. I have tested this on windows, and I want to add a couple of lines to explain it.

### PR DESCRIPTION
**Pull request by nadaa**
Add info to complete adb installation for windows.
Closes #189

### CODE REVIEW

File: contributing.rst [hunk 1]
```diff
@@ -192,7 +192,7 @@ We would also appreciate it if you would keep that file organized by placing the
 Android Tools
 ~~~~~~~~~~~~~~~~~
 
-Some testing and documentation tasks (including :ref:`making screenshots from ODK Collect <screenshots>`) require the :command:`adb` (`Android Debug Bridge <https://developer.android.com/studio/command-line/adb.html>`_) command line tool.
+Some testing and documentation tasks (including :ref:`making screenshots from ODK Collect <screenshots>`) require the :command:`adb` (`Android Debug Bridge <https://developer.android.com/studio/command-line/adb.html>`_) command line tool. You can get the ODB tool either by installing Android studio, or as a standalone SDK tool, as shown bellow.
```
Comment (adammichaelwood) [inline contributing.rst]: typo -- "get the ADB tool"

File: contributing.rst [hunk 2]
```diff
@@ -201,6 +201,13 @@ Android Studio
 
 On Mac, add the following to your :file:`.bash_profile`
 
+.. note::
+    On Windows, you have to run Andriod Studio once to complete the Installation of ADB. The tool can be found in ``C:\Users\your user name\AppData\Local\Android\sdk\platform-tools``. You need to add it to the environment variable path, use the following command:
```
Comment (adammichaelwood) [inline contributing.rst]: s/Andriod/Android

Use the `:file:` text role for file paths

**Comment by adammichaelwood** [comment]
This is good stuff on ADB -- thanks.

This commit also includes stuff related to a different issue (using Google drive, it seems).
Please do not mix issue work into a single PR.

The way to do that is to start each working session with a new branch from master.

**Comment by adammichaelwood** [comment]
@nadaa 
I resolved the merge conflict.

**Comment by nadaa** [comment]
@adammichaelwood , I do create a new branch for each issue.

File: contributing.rst (hunk 2 above)
Comment (nadaa) [reply contributing.rst]: Done.

**Comment by adammichaelwood** [comment]
I cannot merge this PR as long as it contains the google drive info

**Comment by nadaa** [comment]
Shall I send a new PR?

**Comment by nadaa** [comment]
I have a problem in pushing changes, " it says everything is up-to-date"!

**Comment by adammichaelwood** [comment]
- make sure you have committed locally
 - check your online fork and see if your changes are represented there

**Comment by nadaa** [comment]
Changes don't appear on the fork. I am trying to figure out what happened.

**Comment by adammichaelwood** [comment]
When I get that message, usually on of the following is my problem:

 - I have forgotten to `git add` all the changes
 - I'm not on the branch I think I'm on
 - i am pushing master to origin but not pushing my branch (`git push origin branch-name`)

**Comment by nadaa** [comment]
untitled

**Comment by adammichaelwood** [comment]
And your local is not actually in sync with what you  see on gh?

**Comment by nadaa** [comment]
Yes.

**Comment by adammichaelwood** [comment]
What specific differences are you seeing?

**Comment by nadaa** [comment]
Now, it should be working.

**Pull request merged**